website/pages/docs/disclosure/tabs.mdx

```diff
@@ -410,7 +410,7 @@ function CustomTabs() {
     const styles = useStyles()
 
     return (
-      <StyledTab __css={styles.tab} {...tabProps}>
+      <StyledTab ref={ref} __css={styles.tab} {...tabProps}>
```

Comment:
The `ref` here should be passed to the `useTabs` hook `useTabs({...props, ref})` because we do some merging of all user-defined and internal refs.